From fd6ec0179a4b82f0bd46762252bf52eacc40cd83 Mon Sep 17 00:00:00 2001
From: Madalin Bucur <madalin.bucur@freescale.com>
Date: Fri, 6 Mar 2015 11:22:55 +0200
Subject: [PATCH 895/987] dpaa_eth: fix PFC support

PFC support code was using a removed FQ type.

Signed-off-by: Madalin Bucur <madalin.bucur@freescale.com>
[Yang: Original patch taken from
QorIQ-SDK-V1.8-SOURCE-20150619-yocto.iso]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/net/ethernet/freescale/dpa/dpaa_eth_common.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/freescale/dpa/dpaa_eth_common.c b/drivers/net/ethernet/freescale/dpa/dpaa_eth_common.c
index 7507d96..0096b8e 100644
--- a/drivers/net/ethernet/freescale/dpa/dpaa_eth_common.c
+++ b/drivers/net/ethernet/freescale/dpa/dpaa_eth_common.c
@@ -905,8 +905,7 @@ struct dpa_fq *dpa_fq_alloc(struct device *dev,
 	}
 
 #ifdef CONFIG_FMAN_PFC
-	if (fq_type == FQ_TYPE_TX ||
-			fq_type == FQ_TYPE_TX_RECYCLE)
+	if (fq_type == FQ_TYPE_TX)
 		for (i = 0; i < fqids->count; i++)
 			dpa_fq[i].wq = i / dpa_num_cpus;
 	else
-- 
1.9.1

